packages/lit-dev-content/src/components/litdev-aside.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {css, html, LitElement, nothing} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {greenCheckIcon} from '../icons/green-check-icon.js';
+import {redXIcon} from '../icons/red-x-icon.js';
+import {yellowBangIcon} from '../icons/yellow-bang-icon.js';
+import {blueInfoIcon} from '../icons/blue-info-icon.js';
+
+export type AsideVariant = 'positive' | 'negative' | 'warn' | 'info';
+
+@customElement('litdev-aside')
+export class LitDevAside extends LitElement {
+  static styles = css`
+    :host {
+      color: currentColor;
+      display: block;
+      margin: 1em 0;
+    }
+
+    aside {
+      display: flex;
+      border-style: solid;
+      border-width: 1px;
+      border-color: var(--lit-dev-aside-border-color, #ccc);
+      padding: 1em 1em 1em 0em;
+    }
+
+    slot {
+      display: block;
+      flex-grow: 1;
+    }
+
+    svg {
+      width: 1.5em;
+      margin-inline: 1em;
+    }
+
+    :host(:not([no-header])) ::slotted(:first-child) {
+      font-weight: bold;
+    }
+
+    ::slotted(*) {
+      margin: 0;
+    }
+  `;
+
+  @property({type: String})
+  type: AsideVariant = 'info';
+
+  @property({type: Boolean, reflect: true, attribute: 'no-header'})
```

Comment:
Is there any reason for this to reflect?

Reply:
the attribute is used for styling. Should make all efforts to make sure that it is applied, no? TBH it doesn't really need to be a reactive property, but in some future cases lit-plugin may error / warn.

WDYT?

Reply:
Oh sorry, I missed that it was used in the css. Makes sense.